src/mon/MonCommands.h

```diff
@@ -175,6 +175,12 @@ COMMAND("auth get-or-create " \
 	"name=caps,type=CephString,n=N,req=false", \
 	"add auth info for <entity> from input file, or random key if no input given, and/or any caps specified in the command", \
 	"auth", "rwx", "cli,rest")
+COMMAND("auth fs " \
```

Comment:
I'd prefer this to be something like `fs authorize` as well. And there is a precedent. The most obvious would be `pg map`, which is handled by the OSDMonitor.
Basically, you'll have to add an exception to `Monitor::handle_command()`, where it checks `if (module == "mds" || module == "fs")`, and add a new `|| prefix == "fs authorize"` (or whatever ends up being the name of the command.
For correctness' sake, you'll have to keep the `COMMAND`'s third field, which now reads `auth`, as `auth`. This is used to validate caps during `Monitor::_allowed_command()`.

Let me know if you bump into issues, and I'll be happy to help you out.